### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by yoannsculo**
- align French translation order with English to ease future work
- add missing translation
- clean up translation file (fix small typo/mistakes)

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
-    <string name="menuPrevious">Article précédent</string><string name="menuDelete">Supprimer</string>
+    <string name="menuPrevious">Article précédent</string>
+    <string name="menuDelete">Delete</string>
     <string name="menuOpenOriginal">Ouvrir l\'original</string>
+    <string name="menuCopyOriginalURL">Copier l\'URL originale</string>
+    <string name="menuTTS">Synthèse vocale</string>
+    <string name="ttsSpeed">Vitesse de lecture</string>
+    <string name="ttsPitch">Hauteur de la voix</string>
+    <string name="ttsVolume">Volume</string>
+    <string name="ttsLanguage">Choisir la Langue</string>
```
Comment (nicosomb) [inline app/src/main/res/values-fr/strings.xml]: `Langue` -> `langue`

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
+    <string name="ttsVoice">Choisir la voix</string>
+    <string name="ttsAutoplayNextArticle">Lecture automatique de l\'article suivant</string>
+    <string name="ttsError">Erreur de synthèse vocale</string>
+    <string name="ttsFastRewind">Retour Rapide</string>
```
Comment (nicosomb) [inline app/src/main/res/values-fr/strings.xml]: -> `Retour rapide`

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -1,19 +1,35 @@
...
+    <string name="ttsPlayPause">Lecture / Pause</string>
+    <string name="ttsFastForward">Avance Rapide</string>
```
Comment (nicosomb) [inline app/src/main/res/values-fr/strings.xml]: `Avance rapide`

**Comment by yoannsculo** [comment]
Thank you @nicosomb I hadn't seen them. I just pushed a new commit to my french_translation branch.

**Comment by Strubbl** [comment]
Thanks for the PR and the review. 

I am not sure if the order is kept when using weblate ( https://hosted.weblate.org/projects/wallabag/android-app/ ) for translations.

**Comment by yoannsculo** [comment]
Thanks!
@Strubbl It thought weblate was adding new values at the end of the file. But I just checked commit history, and I'm not so sure anymore.

**Pull request merged**